products/green_fast_track/seeds/_seeds.yml

```diff
@@ -32,3 +33,36 @@ seeds:
           - not_null
           - accepted_values:
               values: ["TRUE", "FALSE"]
+
+  - name: expected_green_fast_track_pilot_projects
+    description: Mock pilot projects and their expected outputs
+    config:
+      # empty columns default to integer type
+      column_types:
+        bbl: text
+        Unpermitted Industrial Sources: text
+        Vent Towers: text
+        (E) Designations - Air Quality: text
+        (E) Designations - Noise: text
+        Arterial Highways: text
+        (E) Designations - HazMat: text
+        Special Coastal Risk Districts: text
+        NYC Parks properties: text
+        POPS: text
+        Waterfront Public Access Areas: text
+        Publicly Owned Waterfront: text
+        NYS Parks Property: text
+        Federal Parks property: text
+        geom: geometry(Geometry, 2263)
```

Comment:
wait, I thought dbt doesn't let you check geom data types. Is that not true?

Reply:
this is setting a column type ([docs](https://docs.getdbt.com/reference/resource-configs/column_types)) not testing it

this column is used in the `dbt_utils.equality` test, but that isn't the same as the data contracts we tried that were checking column types and couldn't handle the geometry type

maybe if we try a data contract using something like what I did here with `geometry(Geometry, 2263)` to be more explicit with the type it would work??